get-to-know-syskit-point/generate-reports/licenses-reports.md

```diff
@@ -1,114 +1,157 @@
 # Licenses Reports
 
-Reports in this section help manage your Microsoft 365 licenses. Use them to discover assigned, unassigned or inactive Microsoft 365 licenses, check their overall cost. You can also use these reports to identify underutilized licenses to drive adoption and reduce the cost of the licenses by assigning less expensive ones to users.
+Reports in this section **help manage your Microsoft 365 
+licenses**. 
+
+In this article you can locate information on the following license reports:
+* [Licenses Overview Report](#licenses-overview-report)
+* [Licenses Usage by Service Report](#licenses-usage-by-service-report)
+* [License Distribution](#license-distribution)
+* [Inactive Licenses Report](#inactive-licenses-report)
+
+Use them to **discover assigned, unassigned or inactive Microsoft 365 licenses** and check their overall cost. 
+
+These reports can also be used to **identify underutilized licenses to drive adoption and reduce the cost of the licenses** by assigning less expensive ones to users.
 
 ## Licenses Overview Report
 
-Licenses Overview report helps you to discover the assigned, unassigned and inactive Microsoft 365 licenses on your Microsoft 365 tenant. It also shows the overall cost of all the licenses you currently have purchased from Microsoft. 
+The **Licenses Overview report** helps discover the assigned, unassigned and inactive Microsoft 365 licenses on your Microsoft 365 tenant. 
+It also **shows the overall cost of all the current licenses** you've purchased from Microsoft. 
 
-Use this report to help yourself calculate future license costs.
+This report can be **used to help calculate future license costs**. 
 
 ### Generate the Licenses Overview Report
 
-- From the SysKit Point home screen, click on the Reports tile.
-- In the upper left corner, click on the filter and select Licenses filter.
-- Click on the Licenses Overview report to generate the report.
+ * **Click on the Reports tile**; it is located on the SysKit Point home screen.
```

Comment:
svuda u članku izbaciti 'Click on the ..' frazu. Koristimo svuda 'Click the ..'
interakcije sa UI su pokrivene u MS style guideu pa proući kad stigneš:
https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui